compiler/code-gen/includes.cpp

```diff
@@ -62,6 +62,10 @@ void IncludesCollector::add_function_body_depends(const FunctionPtr &function) {
     }
   }
 
+  for(auto const_var : function->explicit_const_var_ids) {
```

Comment:
Let's try to avoid `auto` for `VarPtr`, `FunctionPtr` and similar. The code becomes much less understandable